docs/source/datasetinsights.rst

```diff
@@ -7,7 +7,6 @@ datasetinsights
 
    datasetinsights.commands
    datasetinsights.datasets
-   datasetinsights.estimators
    datasetinsights.evaluation_metrics
```

Comment:
This should be removed as `datasetinsights.evaluation_metrics` are removed from the package.